src/test/java/uk/ac/cam/cl/dtg/segue/comm/EmailManagerTest.java

```diff
@@ -311,12 +309,12 @@ public final void sendFederatedPasswordReset_checkForTemplateCompletion_emailSho
 
     } catch (ContentManagerException e) {
       e.printStackTrace();
-      Assert.fail();
+      Assertions.fail();
       log.debug(e.getMessage());
```

Comment:
Get rid of this, please, this is not needed, it's better to simply pass the exception to `fail`.